packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import {css, StyleSheet} from "aphrodite";
+import * as React from "react";
+import {Popper} from "react-popper";
+
+import visibilityModifierDefaultConfig from "../util/visibility-modifier.js";
+
+import type {PopperChildrenProps} from "react-popper";
+import type {Placement} from "../util/types.js";
+
+type Props = {|
```

Comment:
Thanks for taking the plunge and using exact types.  I haven't been pressing people on it since the version of flow we're on has issues with spreading props.  Admittedly we don't do that that often.

Reply:
It's habit now and that's worked out great. The cases where we don't want exact make themselves apparent really quickly.